EtlPipelineBlobTrigger/locations.py

```diff
@@ -0,0 +1,22 @@
+import xmltodict
+import xml.etree.ElementTree as ET
+
+
+class Locations:
+    """Provides lookup of raw location data based on UKPRN and LOCID"""
+
+    def __init__(self, root):
+        """Build the locations lookup table
+
+        Locations are unique on UKPRN and LOCID
+        """
+
+        self.lookup_dict = {}
```

Comment:
I haven't seen the use of self since my perl days, i wouldn't have thought to do it like this, nice! - you can ignore this, it's just a passing comment :)